chargeOclock/Source/Models/Client.swift

```diff
@@ -14,8 +14,8 @@ protocol ClientSettable {
 
 class Client {
 
-	var id: Int = 0
-	var isDirty: Bool = false
+	var id: Int? = nil
+	var isDirty: Bool!
```

Comment:
you do realise that nothing forces anyone to initialise anything here, right?
the only thing you introduced with this is an optional that doesn't "code complete" into an obvious optional, but might be nil... which is pretty much a recipe for disaster